addons/website_sale_slides/views/slide_channel_views.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="slide_channel_view_form" model="ir.ui.view">
+            <field name="name">slide.channel.form</field>
```

Comment:
Guidelines for name: slide.channel.view.form.inherit.sale :) . Xml id is correct, just name.